### PR TITLE
fix isComponentRootNode method for react 15.*

### PR DESCRIPTION
**Pull request by Mavrin**
Hi! I  have fixed root detection for react 15 and added test for it.
I add chai assertion because on windows platform i have other path to file.
I noticed than tests on react 14 isn't run on travis now. I think is not good.
It would be cool if you have added style checker for this project.

### CODE REVIEW

**Comment by Mavrin** [comment]
@lahmatiy Could you please merge or review this request?

File: src/api-adapter/react/create-api.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ function getAdditionalInstanceInfo(element) {
   }
 
   if (info && Array.isArray(info.decorators)) {
-    decorators = info.decorators.map(function(decorator, idx) {
+    decorators = info.decorators.map(function (decorator, idx) {
```
Comment (lahmatiy) [inline src/api-adapter/react/create-api.js]: no space betwee `function` and `(` please

File: src/api-adapter/react/create-api.js [hunk 2]
```diff
@@ -230,37 +230,45 @@ function getNestedComponentNodeLocation(node) {
...
       return reactApi.ComponentTree.getNodeFromInstance(element);
     };
+
+    isComponentRootNode = function (node) {
+      var element = getReactElementByNode(node);
+      if (!element) {
+        return false;
+      }
+      return getInstanceRootNode(element) === node;
+    }
```
Comment (lahmatiy) [inline src/api-adapter/react/create-api.js]: semicolon please

**Comment by lahmatiy** [comment]
Sorry about delay. All is good except code style...

> It would be cool if you have added style checker for this project.

Yep, will do

**Comment by lahmatiy** [comment]
Code style linting added a763661eebf53fe80a20280717136a941bdc34db

**Comment by lahmatiy** [comment]
I missed you'd update PR :(
Thank you!

**Comment by Mavrin** [comment]
Cool!

сб, 2 июля 2016, 13:42 Roman Dvornov <notifications@github.com>:

**Pull request merged**